charts/opensearch-cluster/values.yaml

```diff
@@ -68,6 +68,9 @@ cluster:
       # -- Enable cluster monitoring
       enable: false
 
+      # -- Additional labels to apply to the generated ServiceMonitor
```

Comment:
Thanks, I assume this is for OpenSearch and this PR https://github.com/opensearch-project/opensearch-k8s-operator/pull/1002 is for dashbords?

Reply:
Yes, #1002 is for dashboard service resource and this one is for the ServiceMonitor resources. This PR adds missing parts to the chart which are already documented in the [user docs](https://github.com/opensearch-project/opensearch-k8s-operator/blob/main/docs/userguide/main.md?plain=1#L1322-L1323).